### DISCUSSION TITLE
OSX Xcode Support

### DISCUSSION THREAD

**Discussion by vkdevlin** [foonathan/lexy#171] [General]
Hi, I'm developing on OSX Xcode 14.3.1 on OSX Ventura 13.4.1. I added lexy includes to a fresh project and Xcode is immediately complaining. Before I proceed further, is this supported on OSX under Xcode?

**Comment by foonathan** [comment]
It is supported in theory, although CI doesn't test Xcode 14: https://github.com/foonathan/lexy/actions/runs/5661483285/job/15339473229
